packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -152,6 +152,10 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
 
   const { backgroundColor: _, ...formItemStyles } = values?.styles ?? {};
 
+  if (isEmpty(options)) {
```

Comment:
Maybe check for `null` not just emptiness. Sometimes an empty list is a valid use case.